app/views/challenges/show.html.erb

```diff
@@ -1,3 +1,92 @@
-<h1>Challenges#show</h1>
-<p>Find me in app/views/challenges/show.html.erb</p>
+<div class="col-12 mt-3">
+  <%= image_tag("idea.png", size: "48x48") %>
+</div>
+
+<h1><%= @challenge.mode.capitalize %> Mode</h1>
+
+<div id="clockdiv">
+  <div>
+    <span class="days"></span>
+    <div class="smalltext">Days</div>
+  </div>
+  <div>
+    <span class="hours"></span>
+    <div class="smalltext">Hours</div>
+  </div>
+  <div>
+    <span class="minutes"></span>
+    <div class="smalltext">Minutes</div>
+  </div>
+  <div>
+    <span class="seconds"></span>
+    <div class="smalltext">Seconds</div>
+  </div>
+</div>
+
+
+<div class="container">
+  <div class="row text-center m-3">
+    <div class="col-12">
+      <h1> Beat <%= @challenge.opponent.user.username %>! </h1>
+    </div>
+  </div>
+</div>
+
+<div class="row text-center">
+  <img class="challenge-avatar mx-auto d-block"  src="https://images.unsplash.com/photo-1541364983171-a8ba01e95cfc?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60" alt="">
+</div>
+
+<div class="row justify-content-center">
+  <!-- <div class="btn btn-purple text-center mx-6"> -->
+          <%= link_to 'Finished - Ready To Serve!', challenge_serve_path(@challenge), class:'btn btn-purple'  %>
+    <!-- <p>Finished - Ready To Serve!</p> -->
+  </div>
+</div>
+<!-- </body>
+</html>
+-->
+<script>
```

Comment:
This is solid so not necessary, but just to flag: do we want to ask about moving this into a JS file rather than having it all in one place? maybe it's unnecessary and overcomplicating

Reply:
agreed. I had spoken with a certain ta for help and I walked away with a few more questions than answers so I kinda glossed over doing that...